### DISCUSSION TITLE
Kudos for dbmate!

### DISCUSSION THREAD

**Discussion by marco-m** [amacneil/dbmate#561] [Support]
Hello Adrian,

I stumbled on dbmate from sqlc.

Reading the README, I was wondering: "Can I use it as a library?", and it was enough to keep reading to find the Yes answer.
Then just after there is the section "Embedding migrations", another plus!

This tells me that you really thought out what the requirements are for a db migration tool.
And for sure, I appreciate the lightweight and the "Does not try to upsell you on a SaaS service" :-)

So, I just want to tell you Kudos for your work and thanks for sharing, it is a pleasure stumbling upon a well-designed, small and well-explained tool like dbmate. I still have to try it out, but for sure it is in my very short shortlist.

**Comment by amacneil** [comment]
thanks, appreciate the feedback 😄 

also thanks to @dossy who has been helping maintain it!
